include/bluetooth/audio/audio.h

```diff
@@ -918,6 +953,12 @@ struct bt_audio_stream {
 	void *user_data;
 };
 
+/** @brief Audio Capability type */
+enum bt_audio_pac_type {
+	BT_AUDIO_SINK = 0x01,
+	BT_AUDIO_SOURCE = 0x02,
+};
+
```

Comment:
I know you move the code only, but I saw this is used for ASE direction as well. Maybe the enum name should be different then.

Reply:
I agree. Do you have any suggestions? `bt_audio_type` seems a bit too generic for my taste. `bt_audio_ep_type` perhaps?

Reply:
Let's fix this in a new issue: https://github.com/zephyrproject-rtos/zephyr/issues/43693